modules/react/button/stories/button/Button.stories.mdx

```diff
@@ -47,50 +46,38 @@ when you need to place a Primary Button on a dark or colorful background such as
 
 <ExampleCodeBlock code={PrimaryInverse} />
 
-#### Component API
-
-<SymbolDoc name="PrimaryButton" fileName="/react/" hideHeading hideDescription headingStart={4} />
```

Comment:
Note the difference in the Button docs. Instead of displaying the Component API section in the Usage section underneath each component, we now display a single Component API section for all the buttons.

My reasoning for this:

1. Button is an exception (see the [Component Families](https://github.com/Workday/canvas-kit/pull/2310/files#diff-5eed4904a92938d88cb229dd58f9bc3e48fea2a1aab1b7e0944261b680e5959eR134) section in the Documentation Guidelines). Most of our component docs only document a single top-level component whereas Button documents multiple top-level components. This updated structure brings Button more in line with the structure of our other component docs.
2. Although the props tables for each button are no longer colocated with the corresponding example for that button, I think the overall Button doc reads more clearly now that props tables no longer interrupt the flow between examples for each button.

Reply:
I still think since buttons is the only family mdx file we have, we should just break it out into each mdx file. We do this for text component. If we had a unified `<Button/>` component then that would make sense. But since this is the only outlier I say we break them out into their own mdx files. That would be however, that maybe on the side we break it out to `primaryButton` `secondary` pages? I see the con of this is as well. Just posting my thought here but not a blocker for this PR.

Reply:
After discussion with the rest of the team, it sounds like we may break the Button components into their own MDXes in a future PR.